swift/third_party/swift-llvm-support/BUILD.swift-prebuilt.bazel

```diff
@@ -23,6 +29,9 @@ cc_library(
             "-lpthread",
             "-ldl",
         ],
+        "@platforms//os:macos": [
+            "-L/usr/lib/swift",
```

Comment:
will this path be on all macOS installations?

Reply:
I'd expect so, yes.